apps/linkedin-to-notion/src/components/LinkedInNotionSidePanel/LinkedInNotionSidePanelContent.tsx

```diff
@@ -0,0 +1,37 @@
+import logo from 'data-base64:~assets/icon.png';
+import cssText from 'data-text:~style.css';
+import { ButtonPrimary, Heading2, IFramedSidePanel } from 'design-system';
+import { createElement } from 'react';
+
+import './../../../style.css'; // for the font to load
+
+export const getIFrameStyle = () => {
+  return createElement('style', {}, cssText);
+};
```

Comment:
And this is where the *inner* styling of the iframe occurs, like the layout of the buttons inside. Still correct?

Reply:
Yes 👌

Reply:
And I'm injecting it differently because I'm using react portals to inject stuff in the iFrame and it needs to be wrapped in a react element